certified-connectors/Africa's Talking Airtime/apiDefinition.swagger.json

```diff
@@ -0,0 +1,303 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Africa's Talking Airtime",
+    "description": "Distribute virtual, pinless airtime to your users instantly.",
```

Comment:
@crispinkoech This description looks very short. Please provide more detailed description which should should describe concisely the main purpose and value offered by your connector.

Refer - [Documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#submission-requirements)